app/src/main/kotlin/com/thefuntasty/androidprojecttemplate/TestService.kt

```diff
@@ -0,0 +1,12 @@
+package com.thefuntasty.androidprojecttemplate
+
+import android.app.Service
+import android.content.Intent
+import android.os.IBinder
+
+class TestService : Service() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/android-project-template/blob/b0765a4c13fb0e70984828239cb03f4d8d6ea3cf/app/src/main/kotlin/com/thefuntasty/androidprojecttemplate/TestService.kt#L7"></span>The <code>&lt;service&gt; com.thefuntasty.androidprojecttemplate.TestService</code> is not registered in the manifest</td>
    </tr>
  </tbody>
</table>